modules/logging/formattedencoder.go

```diff
@@ -0,0 +1,128 @@
+// Copyright 2015 Matthew Holt and The Caddy Authors
```

Comment:
Since this is going into a different repo, and you wrote the whole thing, you can claim the copyright for it I think.

Reply:
It says `and The Caddy Authors` so it seems inclusive of anyone who contributed to the body of code.